pwnlib/file/filepointer.py

```diff
@@ -176,7 +176,11 @@ def __setattr__(self,item,value):
     def __repr__(self):
         structure=[]
         for i in self.vars_:
-            structure.append(" %s: %#x" % (i, getattr(self, i)))
+            e = getattr(self, i)
+            if isinstance(e, bytes):
+                structure.append(" %s: %s" % {i, e})
```

Comment:
This will reverse the pair nondeterministically with 50% prob. You probably want a tuple.

Reply:
I see. That's my fault.